lib/constants.ts

```diff
@@ -119,10 +119,6 @@ export const OIDC_VERIFY_SSL = true;
 export const OIDC_VERIFY_SIGNATURE = true;
 // This defaults to the APIGW url but if you're using custom DNS you should set this to that
 export const OIDC_REDIRECT_URI = '';
-// Optional system banner which will be displayed at the top and the bottom of MLSpace
```

Comment:
Let's leave these in for now. If I remember correctly, in the dev sync we discussed keeping these for now and marking them as deprecated. We'll keep them for the next release, then remove them for the release following that.

We're good to keep them remove from `restApi` since we don't need to deploy them, but we should keep them in `configTypes` since we'll want to support them in the magic config still as well. 

What we should do with these values is update `lib/utils/initialAppConfig.ts ` so that the default system banner settings are based on these.

```
'SystemBanner': {'M': {
                'isEnabled': {'BOOL':  mlspaceConfig.SYSTEM_BANNER_TEXT !== ''}, 
                'text': {'S': mlspaceConfig.SYSTEM_BANNER_TEXT}, 
                'textColor': {'S': mlspaceConfig.SYSTEM_BANNER_TEXT_COLOR}, 
                'backgroundColor': {'S': mlspaceConfig.SYSTEM_BANNER_BACKGROUND_COLOR}
            }}, 
```